src/boards/include/boards/0xcb_helios.h

```diff
@@ -73,7 +73,7 @@
 #endif
 
 // board has 16M onboard flash
-// pico_cmake_set_default PICO_FLASH_SIZE_BYTES = (16 * 1024 * 1024)
+pico_cmake_set_default(PICO_FLASH_SIZE_BYTES, 16 * 1024 * 1024)
```

Comment:
Now that this "looks" like a proper macro, is there a risk that people might expect it to behave like a proper macro? E.g. if somebody did:
```c
// board has 16M onboard flash
#define DEFAULT_MEMORY_SIZE (16 * 1024 * 1024)
pico_cmake_set_default(PICO_FLASH_SIZE_BYTES, DEFAULT_MEMORY_SIZE)
#ifndef PICO_FLASH_SIZE_BYTES
#define PICO_FLASH_SIZE_BYTES DEFAULT_MEMORY_SIZE
#endif
```
would it work as expected?